### ISSUE TITLE
Error when trying the SDK Generation Example

### ISSUE THREAD

**Issue by rschmidtz** [aws/chalice#465]
Hi, I was following the example to generate the SDK: [link](http://chalice.readthedocs.io/en/latest/topics/sdks.html), and I got this error:

`An error occurred (NotFoundException) when calling the GetSdk operation: Invalid stage identifier specified`

Comments?

**Comment by stealthycoin** [comment]
Which version of chalice was this? Works for me on the most current reason.

**Comment by rschmidtz** [comment]
chalice --version

`chalice 1.0.0`

**Comment by stealthycoin** [comment]
Can you try adding `--debug` as the first argument after `chalice`. Something like:
`chalice --debug generate-sdk foo`.  Probably an incorrect api-gateway-stage name was configured or is being passed into the generate-sdk. I think this may be fixed with #745 but I am not positive. I can't reproduce this without some more details from you about what stages you have and what command exactly you are running.

**Comment by jamesls** [comment]
Closing due to inactivity.  I'm not able to repro but if you can share a set of steps that demonstrate the issue we can reopen and investigate.  Here's what I tried:


```
$ chalice new-project testsdk
$ cd testsdk/
$ chalice deploy
Creating role: testsdk-dev
Creating deployment package.
Creating lambda function: testsdk-dev
Initiating first time deployment.
Deploying to API Gateway stage: api
https://rest-api-id.execute-api.us-west-2.amazonaws.com/api/
$ http https://rest-api-id.execute-api.us-west-2.amazonaws.com/api/
HTTP/1.1 200 OK
...

{
    "hello": "world"
}

$ chalice generate-sdk js-sdk
$ ls js-sdk/
README.md     apigClient.js lib
```

I also tried it with the sample app from the [docs page](http://chalice.readthedocs.io/en/latest/topics/sdks.html) and I'm also able to generate an SDK.
